alerts/forward.libsonnet

```diff
@@ -51,6 +51,32 @@
               message: 'CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of forward requests to {{ $labels.to }}.',
             },
           },
+          {
+            alert: 'CoreDNSForwardHealthcheckFailureCount',
+            expr: |||
+              rate(coredns_forward_healthcheck_failure_count_total[10s]) > 3
+            ||| % $._config,
+            'for': '30s',
```

Comment:
I would rather make this be 10 minutes. Any reason why 30 seconds?

Reply:
@lilic my thinking is that if this rate is achieved, it can be a pretty significant issue for apps that require the upstream for name resolution. However, I'll change to 10m based on your guidance.

Reply:
Commit `5cbc692` addresses your comment here.